js/interface.js

```diff
@@ -1929,7 +1935,6 @@ var DynamicLists = (function() {
       $filterAccordionContainer.append($newPanel);
 
       if (data.logic === 'empty' || data.logic === 'notempty') {
-        $newPanel.find('.panel-title-text .value, #value-dash, #filter-value-type').addClass('hidden');
```

Comment:
Why is this removed? If you look at [the previously approved PR](https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/473), the `interface.js` file had this line https://github.com/AndrRyaz/fliplet-widget-dynamic-lists/blob/7eb19e2dc9f7b5c0de46d1b57945f6d9c70f0a1e/js/interface.js#L1940

Reply:
@tonytlwu, when i created a new branch from base master and update them from upstream, this changes was already implemented, but your right this line should be, fixed